### DISCUSSION TITLE
export audio clips with correct date and timestamp in filenames

### DISCUSSION THREAD

**Discussion by RanSchols** [Mattk70/Chirpity-Electron#128] [General]
As a nocmigger I a recently started experimenting with your fantastic tool. Thanks for you effords! 
Setting the right starting time of a night-long recording with "Amend/ submit File Start Time" is of great help to see/get the real time a bird was passing during the night. But when I want to save and export the audioclips it seems to write the original timestamp in the filenames and not the real/right times. Also a audioclip after midnight does get the the wrong date (the startingdate). Is this a bug or am i missing a certain setting in Chirpity?
Thanks Ran, The Netherlands

**Comment by Mattk70** [comment]
Hi @RanSchols, when I fixed the timestamp formatting for exported clips in version 1.10.1, I realised these times were all UTC - or the GMT time zone.  This was initially unintentional - and I didn't notice because in the winter months, it is GMT where I live.

I considered changing this to local time, but held back because I was concerned how this would deal with the example of someone that wanted to analyse recordings from different time zones. Thinking about it a little more now, I'm not sure either approach is going to be correct all the time, as the way dates and times are calculated in JavaScript depends on your computer's underlying time zone setting. 

I'll investigate further, but if would involve doing a timezone lookup from the lat / lon that has been set for the file, it may spiral into something quite complicated to do. In which case, I think it would be better for most people if Chirpity was to use local time.

**Reply by RanSchols** [reply]
I guess most birders would like local time stamps in the (saved) audio files (and results in CSV?). Good luck investing the options. Thanks for your quick reply and keep up the good work

**Reply by Mattk70** [reply]
I think I was over thinking it. From 1.10.2, the exported audio filenames will use the same local time as that for the original file.
